lib/devices/Viomi.js

```diff
@@ -178,7 +178,7 @@ class Viomi extends MiioVacuum {
         if (newState["s_area"]) { //TODO: actually not an attribute. Use Job
             this.robotState.upsertFirstMatchingAttribute(new stateAttrs.LatestCleanupStatisticsAttribute({
                 type: stateAttrs.LatestCleanupStatisticsAttribute.TYPE.AREA,
-                value: newState["s_area"] * 1000 //m² to cm²
+                value: newState["s_area"] * 10000 //m² to cm²
```

Comment:
Isn't this exactly the opposite of what's been reported as a bug here? https://github.com/Hypfer/Valetudo/issues/611#issue-714279647

Reply:
This is the same issue and same problem.

The vacuum driver in devices/Viomi.js is seting wrong value when converting m² to cm².
PaskoMoto in #611, fix this value during the presentation of this value in client/home.js, but the problem is within the driver.

I am pretty sure that 1m²  = 10000cm²